emscripten.py

```diff
@@ -1707,16 +1709,21 @@ def math_fix(g):
 STACKTOP = STACK_BASE + TOTAL_STACK;
 STACK_MAX = STACK_BASE;
 HEAP32[%d >> 2] = STACKTOP;
-Runtime.stackAlloc = asm['stackAlloc'];
-Runtime.stackSave = asm['stackSave'];
-Runtime.stackRestore = asm['stackRestore'];
-Runtime.establishStackSpace = asm['establishStackSpace'];
 ''' % shared.Settings.GLOBAL_BASE)
-
+  if settings['BINARYEN_ASYNC_COMPILATION']:
+    funcs_js.append("__ATINIT__.unshift(function() {\n"
```

Comment:
Is there an equivalent of this for asm2wasm? Seems like it should work the same either way?